### DISCUSSION TITLE
Incompatibility pyjwt 2.0.0+ with djangorestframework-simplejwt > 4.7.2

### DISCUSSION THREAD

**Discussion by blv-geoffreyvandercruyssen** [jpadilla/pyjwt#766] [General]
Hello, I only have time for a quick message so no time for creating an issue. When upgrading pyjwt from version 1.7.1 to 2.0.0 and above, and the djangorestframework-simplejwt is version 4.6.0 (or below?), authentication will throw a 401 while executing the init of AccessToken in rest_framework_simplejwt.tokens.py

**Comment by jpadilla** [comment]
this is probably better suited in djangorestframework-simplejwt's discussion/issue tracker
